src/super_gradients/training/transforms/transforms.py

```diff
@@ -775,20 +775,43 @@ class DetectionTargetsFormatTransform(DetectionTransform):
     @resolve_param("output_format", ConcatenatedTensorFormatFactory())
     def __init__(
         self,
-        image_shape: tuple,
+        input_dim: Optional[tuple] = None,
         input_format: ConcatenatedTensorFormat = XYXY_LABEL,
         output_format: ConcatenatedTensorFormat = LABEL_CXCYWH,
         min_bbox_edge_size: float = 1,
         max_targets: int = 120,
     ):
         super(DetectionTargetsFormatTransform, self).__init__()
+        if isinstance(input_format, DetectionTargetsFormat) or isinstance(output_format, DetectionTargetsFormat):
+            raise TypeError(
+                "DetectionTargetsFormat is not supported for input_format and output_format starting from super_gradients==3.0.7.\n"
```

Comment:
I don't get this...

Reply:
I raise an error if they still use DetectionTargetsFormat (the Enum that we used before). It's to make the error explicit for a few versions.
Without this the error that they get if they dont change their old recipe is hard to understand.